samples/routing/fair-distribution/sample.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Fair load distribution
+summary: Implementing fair load distribution for heterogeneous scaled-out endpoints
+component: Core
+reviewed: 2016-06-21
+tags:
+- Routing
+- Distribution
+- DistributionStrategy
+---
+
+
+## Running the project
+
+ 1. Start all the projects by hitting F5.
+ 1. The text `Press <enter> to send a message` should be displayed in the Client's console window. 
+ 1. Hold down `<enter>` for a few seconds to send many messages.
+
+
+### Verifying that the sample works correctly
+
+ 1. Notice more messages are being send to Server.1 compared to Server.2
+ 1. Use a [MSMQ viewing tool](/nservicebus/msmq/viewing-message-content-in-msmq.md) to view the queue contents.
```

Comment:
Prerequisites - MSMQ set up as described in http://docs.particular.net/nservicebus/msmq/